packages/@vuepress/theme-default/src/client/styles/navbar.scss

```diff
@@ -15,10 +15,15 @@
   }
 
   .site-name {
+    display: inline-block;
+    width: calc(100vw - 18rem);
```

Comment:
Obviously because of dynamic contents in navbar, we should never set this to a fixed value.

Reply:
oh，i see. Is it better to limit the navbar-items-wrapper and navbarBrand width with js?
conflict between the two when their too long